test/LockedNORI.t.sol

```diff
@@ -195,35 +215,90 @@ contract LockedNORITest is Test, ERC777ERC1820 {
       address(this)
     );
 
-    bpNori = new TestToken777();
-    assertEq(bpNori.balanceOf(address(this)), SEED_AMOUNT);
+    erc777 = deployMockERC777();
+    assertEq(erc777.balanceOf(address(this)), SEED_AMOUNT);
```

Comment:
FYI, something I learned is that assertions don't work like they do in tests ([assertions in setup() and constructors don't fail tests](https://github.com/foundry-rs/foundry/issues/1291)). Workaround is to do `if(erc777.balanceOf(address(this)) !== SEED_AMOUNT){ revert("Balance !== SEED_AMOUNT");}`

Reply:
Good tip. These are probably unnecessary assertions in the setup anyway.